packages/server/graphql/nestedSchema/nestGitLabEndpoint.ts

```diff
@@ -21,7 +21,8 @@ const defaultExecutor: Executor<{
     controller.abort()
   }, endpointTimeout)
   try {
-    const result = await fetch(`${baseUri}/api/graphql`, {
+    const uri = baseUri || 'https://gitlab.com' // TODO: why is baseUri undefined when adding issues to the Estimate phase?
```

Comment:
When adding an issue to the Estimate phase, the `baseUri` that we get from `context` is null. I've played around with the nestGitLab logic, but I haven't discovered why yet. Possibly an issue with the package?